linters/cmake-format/plugin.yaml

```diff
@@ -0,0 +1,30 @@
+version: 0.1
+tools:
+  definitions:
+    - name: cmake-format
+      runtime: python
+      package: cmake-format
+      shims: [cmake-format]
+      known_good_version: 0.6.13
+lint:
+  definitions:
+    - name: cmake-format
+      files: [cmake]
+      commands:
+        - name: lint
+          run: cmake-format --check ${target}
+          success_codes: [0, 1]
+          read_output_from: stderr
+          output: pass_fail
+        - name: fix
+          formatter: true
+          output: rewrite
+          success_codes: [0, 1]
+          run: cmake-format -i ${target}
+          in_place: true
+      tools: [cmake-format]
+      suggest_if: files_present
```

Comment:
Let's go ahead and change this to `config_present`. `files_present` is probably where we want to be long-term, but we usually dogfood new linters/let them marinate a bit before turning them on for a file basis.